src/Sidekick/App.xaml.cs

```diff
@@ -164,5 +172,33 @@ private void EnsureSingleInstance()
                 Current.Shutdown();
             }
         }
+
+        private void AttachErrorHandlers()
+        {
+            AppDomain.CurrentDomain.UnhandledException += (s, e) =>
+            {
+                var exception = (Exception)e.ExceptionObject;
+                LogUnhandledException(exception);
+            };
+
+            DispatcherUnhandledException += (s, e) =>
+            {
+                LogUnhandledException(e.Exception);
+                e.Handled = true;
+            };
+
+            TaskScheduler.UnobservedTaskException += (s, e) =>
+            {
+                LogUnhandledException(e.Exception);
+                e.SetObserved();
+            };
+        }
+
+        private void LogUnhandledException(Exception ex)
+        {
+            logger.Fatal(ex, "Unhandled exception in application root");
```

Comment:
I'm pretty sure we need to try/catch everything here.

Reply:
Serilog will never throw an uncaught exception when logging, no need to try/catch.